Dev/issue 13530 b5 themes uppy data params title

sbreker opened the pull request:
This commit updates the Uppy data elements to 'kebab-case'. 

A bug was introduced when porting this script to B5 whereby image title
updates were not being saved because the assignment of this.$element
was changed. Revert saving $(element) to this.$element and change the 
data node to use a different class property.

sbreker committed:
Uppy multifile upload title update issue

A bug was introduced when porting this script to B5 whereby image title
updates were not being saved.

- Relocated data elements to a div that includes all the nodes that
multiFileUpload.js needs access to. This was causing issues selecting
the data attributes and the form buttons etc.
- Changed the selected class name to kebab case as this was preventing
correct selection.
- update the Uppy data elements to use 'kebab-case'.

jraddaoui commented on plugins/arDominionB5Plugin/js/multiFileUpload.js:
```diff
@@ -4,7 +4,8 @@
   function MultiFileUpload (element)
   {
-    this.$element = $(element).find('div.multiFileUpload');
+    this.$multiFileUpload = $(element).find('div.multiFileUpload');
+    this.$element = $(element);
```
I don't understand why do you need two vars here ... based on line 386, you already have the correct element and line 8 should be enough to use `this.$element` for everything ... right?

sbreker replied:
I have fixed this underlying issue allowing me to remove the second variable.

jraddaoui reviewed (commented):
Thanks @sbreker!

sbreker commented:
Thanks for your review @jraddaoui!  I have pushed a new commit that better addresses the root issues affecting saving the titles correctly.

The selector in the document ready function was not selecting correctly due to kebab-case issues it seems. I have renamed this. Secondly, I have moved the data elements to a higher div on the page so that the full form is selected - the js needs access to buttons AND the data elements.

jraddaoui commented on plugins/arDominionB5Plugin/js/multiFileUpload.js:
```diff
@@ -384,7 +383,8 @@
   {
-    var $node = $('.multiFileUpload');
+    var $node = $('.multifileupload-form');
+    //console.log($node.data('multifileupload-i18n-some-files-failed-error'));
```
Ups ...

sbreker replied:
Removed...

jraddaoui reviewed (approved):
Awesome, thanks @sbreker!
